presto-docs/src/main/sphinx/presto_cpp/limitations.rst

```diff
@@ -0,0 +1,44 @@
+======================
+Presto C++ Limitations
+======================
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+General Limitations
+===================
+
+The C++ evaluation engine has a number of limitations:
+
+* Not all built-in functions are implemented in C++. Attempting to use unimplemented functions results in a query failure. For supported functions, see `Function Coverage <https://facebookincubator.github.io/velox/functions/presto/coverage.html>`_.
+
+* Not all built-in types are implemented in C++. Attempting to use unimplemented types will result in a query failure. 
+
+* Certain parts of the plugin SPI are not used by the C++ evaluation engine. In particular, C++ workers will not load any plugin in the plugins directory, and certain plugin types are either partially or completely unsupported.  
+
+  * ``PageSourceProvider``, ``RecordSetProvider``, and ``PageSinkProvider`` do not work in the C++ evaluation engine.
+
+  * User-supplied functions, types, parametric types and block encodings are not supported.
+
+  * The event listener plugin does not work at the split level.
+
+  * User-defined functions do not work in the same way, see **LINK**.
```

Comment:
I need a link, a list, or another solution to this placeholder `**LINK**` text. @tdcmeehan, please follow up on this when you can.

Reply:
I just remembered, that I think this link should be here: https://github.com/prestodb/presto/blob/5c8fc862c7170f33aa2dac2c7124464fad523191/presto-docs/src/main/sphinx/prestissimo/prestissimo-features.rst#remote-function-execution

Reply:
`User-defined remote functions do not work in the same way, see here: https://github.com/prestodb/presto/blob/5c8fc862c7170f33aa2dac2c7124464fad523191/presto-docs/src/main/sphinx/prestissimo/prestissimo-features.rst#remote-function-execution`

Reply:
Thanks! Link makes sense in context. Added link, test link in local build and it works, pushed new commit. Will squash commits before merge.